File: Beginner_Projects/Patterns/Alphabats/Python/O.py
for row in range(5):
    for col in range(6):
        if (((row * col == 0 and (col + row != 0) and col != 5 and row != 4)) or (col == 5 and row != 0 and row != 4) or (row == 4 and col != 0 and col != 5)):

            print("*", end=" ")
        else:
            print(end="  ")
    print()
input()

File: Beginner_Projects/Patterns/Alphabats/C++/O.cpp
#include <iostream>
#include <conio.h>
using namespace std;

int main()
{
    for (int row = 0; row < 5; row++)
    {
        for (int col = 0; col < 6; col++)
        {
            if (((row * col == 0 && (col + row != 0) && col != 5 && row != 4)) || (col == 5 && row != 0 && row != 4) || (row == 4 && col != 0 && col != 5))
            {
                cout << "*";
            }
            else
            {
                cout << " ";
            }
        }
        cout << endl;
    }
    getch();
}


//Created by Salfi Hacker Mansoor Bukhari